SDK/Examples/CppDynamicNew/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+#[[++
```

Comment:
We should rename CppDynamicNew and CppNew to a symbolizing name. Do you have any suggestions?

Reply:
We can move this to another folder CPackExamples or ExamplesUsingCPack and have those there. Does this sound okay to you?